typescript/codepipeline-build-deploy/lib/codepipeline-build-deploy-stack.ts

```diff
@@ -1,10 +1,12 @@
 import * as cdk from "aws-cdk-lib";
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
Is there a reason you have this?  This refers to the old cdk v1 lib. and we also have this defined correctly on line 13 of this file.